packages/waku/src/lib/plugins/vite-plugin-rsc-entries.ts

```diff
@@ -17,6 +17,8 @@ const CONFIG_FILE = 'waku.config.ts'; // XXX only ts extension
 export function rscEntriesPlugin(opts: {
   basePath: string;
   rscBase: string;
+  middleware: string[];
```

Comment:
I can follow up this PR with type suggestions for the waku supported plugins if you're open to it. It would still allow any string, but suggest the built in ones.

Reply:
Can you highlight your idea with a few lines of code?

Reply:
playground: https://tsplay.dev/we3Xem

<img width="720" alt="image" src="https://github.com/user-attachments/assets/7fd650c8-aa8b-4e6a-b3fc-852e995e7443" />

Reply:
Thanks. I see this doesn't break anything. Not sure how much it helps because it's very unlikely to people to type it. (Besides, this isn't a fix for the regression.)

I don't think we need it at least in this PR.